nixos/modules/installer/netboot/netboot.nix

```diff
@@ -94,7 +94,7 @@ with lib;
 
     system.build.netbootIpxeScript = pkgs.writeTextDir "netboot.ipxe" ''
       #!ipxe
-      kernel ${pkgs.stdenv.hostPlatform.linux-kernel.target} init=${config.system.build.toplevel}/init initrd=initrd ${toString config.boot.kernelParams}
+      kernel ${pkgs.stdenv.hostPlatform.linux-kernel.target} init=${config.system.build.toplevel}/init initrd=initrd ${toString config.boot.kernelParams} ''${cmdline}
```

Comment:
Please add a comment here why we are adding this.

Reply:
Updated :+1: 

Although I'm not sure if I should default `cmdline` to `config.boot.kernelParams` if `cmdline` is not set, or if it should remain as it is now.